omod/src/main/webapp/pages/metadata/roles/templates/edit.gsp

```diff
@@ -0,0 +1,81 @@
+<h2 ng-hide="role.uuid">${ ui.message("adminui.role.addNewRole.title") }</h2>
+<h2 ng-show="role.uuid">${ ui.message("adminui.role.editRole.title") }</h2>
+
+<form class="simple-form-ui" name="roleForm" novalidate ng-submit="save()">
+    <p>
+        <label>${ ui.message("adminui.role.role") }</label>
+        <input ng-disabled="role.uuid" ng-model="role.name" ng-class="{ adminuiDisabled: role.uuid }" required/>
```

Comment:
I think when editing you don't display the name in a text field, it should just be plain text of the name just like in the old UI, i believe it is the conventional way we do it for fields a user can't edit